.evergreen/config.yml

```diff
@@ -649,6 +654,53 @@ functions:
         script: |
           ${PREPARE_SHELL}
           ${PROJECT_DIRECTORY}/.evergreen/run-mongodb-aws-test.sh
+  run aws auth test AssumeRoleWithWebIdentity with AWS_ROLE_SESSION_NAME set:
```

Comment:
Noticed that this set of changes doesn't have the addition of the `install aws-credential-providers` script that #3556 did - why the discrepancy?

Reply:
In version 4x, the AWS SDK was an optional dependency so it was installed by default. In version five, it is no longer installed by default so we must manually install it.